loader/src/main/java/net/neoforged/fml/loading/ModSorter.java

```diff
@@ -120,6 +120,14 @@ private void sort() {
                 .map(IModInfo::getDependencies).<IModInfo.ModVersion>mapMulti(Iterable::forEach)
                 .forEach(dep -> addDependency(graph, dep));
 
+        FMLConfig.getDependencyOverrides().forEach((id, overrides) -> {
+            for (FMLConfig.DependencyOverride override : overrides) {
+                if (!override.remove()) {
+                    graph.putEdge((ModInfo) modIdNameLookup.get(override.modId()), (ModInfo) modIdNameLookup.get(id));
```

Comment:
What if the mods don't exist? We add null to the graph? :P

Reply:
We will need a check yes, though I'm not sure if it should be a crash or a warning.